packages/ecommerce-ui/components/FormComponents/NumberFieldElement.tsx

```diff
@@ -80,43 +82,69 @@ export function NumberFieldElement<T extends FieldValues>(props: NumberFieldElem
             size={size}
             type='number'
             className={`${textFieldProps.className ?? ''} ${classes.quantity}`}
-            sx={[{ width: responsiveVal(80, 120) }, ...(Array.isArray(sx) ? sx : [sx])]}
+            sx={[
+              {
+                width: responsiveVal(80, 120),
+              },
+              variant === 'standard' && {
+                '& .MuiOutlinedInput-root': {
+                  px: 0,
+                  display: 'grid',
+                  gridTemplateColumns: '1fr auto 1fr',
+                },
+                '& .MuiOutlinedInput-input': {
+                  padding: 0,
+                },
+                '& .MuiOutlinedInput-notchedOutline': {
+                  display: 'none',
+                },
+              },
+              ...(Array.isArray(sx) ? sx : [sx]),
+            ]}
             autoComplete='off'
             InputProps={{
               ...textFieldProps.InputProps,
               startAdornment: (
-                <IconButton
+                <Fab
```

Comment:
What was the reason for switching to Fabs instead of IconButtons?
A Fab seems out of place in this context.